### DISCUSSION TITLE
Welcome to Numpy-Practice Discussions!

### DISCUSSION THREAD

**Discussion by SparklinStar** [SparklinStar/Numpy-Practice#1] [Announcements]
## 👋 Welcome!
  Let's use Discussions as a place to connect with other numpy warriors. We hope that you:
  * Ask questions about any tasks that you are not getting.
  * Share your thoughts about any tasks.
  * Engage with other participants.

  To get started, comment below with an introduction of yourself and tell us about why do you want to learn about numpy.

**Comment by ekramzafar** [comment]
Hello my name is ekram zafar 1st  year student in CSE at jis college of engineering

**Reply by SparklinStar** [reply]
Hey Ekram! Nice to meet you. Go through your PR again. I have asked to create your own folder under which you will do your tasks.

**Comment by chiragnahata** [comment]
Hi,

I am Chirag Nahata, a 1st Year student in CSE (AIML) at JISCE. 

linktr.ee/chiragnahata

**Reply by SparklinStar** [reply]
Hey Chirag! Start by forking this repo and then make a PR after creating a folder with your own name. Under that Folder you will do all the tasks. You can see my folder for reference.

**Comment by dhrubojyotii** [comment]
Hello , 
I am Dhrubojyoti Saha, 1st year student in Information Technology at JISCE

**Reply by SparklinStar** [reply]
Hey Dhrubojyoti! Start by forking this repo and then after creating a folder with your own name make a PR. Under that Folder you will do all the tasks. You can see my folder with the name "Souradip-Pal" for reference.

**Comment by sahinx09** [comment]
### **Hello**,
I am [Sahin Molla](https://linktr.ee/sahinmolla) . 1st year student of **CSE (AIML)** at JISCE

**Reply by SparklinStar** [reply]
Hey Sahin! Start by forking this repo and then after creating a folder with your own name make a PR. Under that Folder you will do all the tasks. You can see my folder with the name "Souradip-Pal" for reference.

**Comment by Gouharhabib** [comment]
Hello my name is Md. Gouhar Habib 1st year student in CST at jis college of engineering

**Comment by piyush182004** [comment]
Hii I am Piyush Kumar Mondal BTech CSE(AIML) first year student at JIS College Of Engineering.

**Reply by Gouharhabib** [reply]
Hello

**Comment by snig-code16** [comment]
Hi, I am Snigdha Ghosh, btech CSE AIML ,1st year at JIS College Of Engineering

**Comment by RAJAT-MITRA** [comment]
Hi, I am Rajat Mitra from B.Tech CSE AI-ML branch, it is my 1st Year at JIS College Of Engineering
